ocp_resources/node_network_configuration_policy.py

```diff
@@ -307,8 +324,9 @@ def wait_for_interface_deleted(self):
     def validate_create(self):
         for pod in self.worker_pods:
             for bridge in self.ifaces:
-                node_network_state = NodeNetworkState(name=pod.node.name)
-                node_network_state.wait_until_up(name=bridge["name"])
+                if "capture" not in bridge["name"]:
```

Comment:
Please explain why

Reply:
When configuring an interface whose name is used with capture, like in: 
``- name: "{{ capture.br1.interfaces.0.bridge.port.0.name }}"
      type: ethernet
      state: up
      ipv4: "{{ capture.br1.interfaces.0.ipv4 }}"
      ipv6: "{{ capture.br1.interfaces.0.ipv6 }}"``
     
py-wrapper checks if interface {{ capture.br1.interfaces.0.bridge.port.0.name }} is up, there's no way to pass/know actual name

Reply:
Please add comment about it in the code.

Reply:
Sure